src/shared/utils/billing.test.ts

```diff
@@ -535,7 +458,7 @@ describe('findTeamPlans', () => {
     const expectedResult = {
       marketingName: 'Team',
       value: Plans.USERS_TEAMY,
-      billingRate: 'yearly',
```

Comment:
sometimes we were using 'yearly' and some other random stuff in our mocks for billingRate sadge